code/index.html

```diff
@@ -2,29 +2,114 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
     <title>Project 3</title>
+    <link rel="stylesheet" href="./style.css"> 
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Abril+Fatface&display=swap" rel="stylesheet">
     <!-- dont forget to add a css file and link it here! -->
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
+   <header class= "img-header">
+      <h1>Your Fake Moustache</h1>
+      <h2>When you are the last babyface in the group</h2>
+   </header> <!-- video or image as a header is cool :) -->
+    
 
-    <!-- Signup form -->
+   <div class="cards">
+    
+      <form id="form" action="https://httpbin.org/anything" method="POST">
+           <div class="form">
+           <p>Sign up here</p>
+            <label>
+             <input type="text" name="message"
+             placeholder="Name" required>
+             </label>
+      
+      <label> 
+        <input type="text" name="message"
```

Comment:
You could write "name" instead on the name label if it's a name someone should type in, so it's more clear in the post file if you also want to have a message at some point. But it all returns nicely so maybe not an issue. 

Nice with the placeholer! Makes the form look tidy!